medusa/process_tv.py

```diff
@@ -334,11 +334,9 @@ def validateDir(self, path, dirName, nzbNameOriginal, failed):
         :param result: Previous results
         :return: True if dir is valid for processing, False if not
         """
-        dirName = ss(dirName)
-
-        IGNORED_FOLDERS = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
+        ignored_folders = [u'.AppleDouble', u'.@__thumb', u'@eaDir']
```

Comment:
If `ignore_folders` is constant it should stay capitalized, should probably be a tuple, and should be moved to the module level so its not recreated upon each call of the function.